debian/changelog

```diff
@@ -1,3 +1,10 @@
+matrix-synapse-py3 (1.12.2ubuntu1) UNRELEASED; urgency=medium
```

Comment:
How come we want `1.12.2ubuntu1` here instead of `1.12.3`? Just to test things before releasing?

Reply:
this is just a placeholder, we don't care what it says except that it says UNRELEASED. When we come to release, the `dch` incantation replaces it with the right version.

Reply:
Ah ok, I assumed something like that would happen.